src/Compilers/Core/CodeAnalysisTest/Diagnostics/CompilationWithAnalyzersTests.cs

```diff
@@ -8,6 +8,7 @@
 using System.Linq;
 using System.Threading;
 using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
```

Comment:
this mostly looks to be on the right track, but this project doesn't seem to have a reference to the C# test base project. You might have to go looking in the compilers core test base project or create a helper just for this file to produce the desired compilation options for each test.